packages/core/src/components/menu/_common.scss

```diff
@@ -38,18 +40,62 @@ $dark-menu-background-color: $dark-gray4 !default;
   }
 
   .pt-dark & {
-    @include dark-menu-item($disabled-selector);
+    @include dark-menu-item($disabled-selector, $hover-selector);
   }
 }
 
-@mixin dark-menu-item($disabled-selector: ".pt-disabled") {
+@mixin dark-menu-item($disabled-selector: ".pt-disabled", $hover-selector: ":hover") {
   color: inherit;
 
+  &:not(#{$disabled-selector})#{$hover-selector} {
```

Comment:
can we plesae avoid this scary specific selector by redefining `background` in the disabled selector?
happens a few times throughout this change.